utils/src/main/java/com/nedap/archie/util/KryoUtil.java

```diff
@@ -1,25 +1,34 @@
 package com.nedap.archie.util;
 
 import com.esotericsoftware.kryo.kryo5.Kryo;
+import com.esotericsoftware.kryo.kryo5.Serializer;
 import com.esotericsoftware.kryo.kryo5.util.Pool;
 
+import java.util.HashMap;
 
 /**
  * Created by pieter.bos on 03/11/15.
  */
 public class KryoUtil {
 
     // Build pool with SoftReferences enabled (optional)
-    private static Pool<Kryo> pool;
+    private static final Pool<Kryo> pool;
+
+    @SuppressWarnings("rawtypes")
+    private static final HashMap<Class, Serializer> serializers = new HashMap<>();
```

Comment:
```suggestion
    private static final HashMap<Class<?>, Serializer<?>> serializers = new HashMap<>();
```
This fixes the warning without suppressing it.